presto-product-tests/conf/tempto/tempto-configuration-for-docker-fileauth.yaml

```diff
@@ -0,0 +1,9 @@
+databases:
+  hive:
+    host: hadoop-master
+  presto:
+    host: presto-master
```

Comment:
redundant, default

however, don't you need to use `presto-master.docker.cluster` here, like other product tests with SSL do?